src/devcards.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import DevCards from './components/DevCards.jsx';
+
+/**
+ * Cards
+ */
+
+// Main card
+export default function devcard(name, doc, body) {
+  devcard.current.push({name, doc, body});
+}
+
+// Noop card
+devcard.off = Function.prototype;
+
+
+/**
+ * Runtime
+ */
+
+// Finding the root
+// TODO: not finished
+function getRoot() {
+
```

Comment:
I guess it would be nice if the `run` function could take the node or its id as argument. But I also like the approach to create the node if nothing is specified or if a convention id is not found.